src/rules/memberAccessRule.ts

```diff
@@ -118,18 +119,35 @@ function walk(ctx: Lint.WalkContext<Options>) {
             return;
         }
 
-        const isPublic = Lint.hasModifier(node.modifiers, ts.SyntaxKind.PublicKeyword);
+        let publicKeyword: ts.Node | undefined;
```

Comment:
:+1:
I added a `getModifier` utility to tsutils. I don't know if I release a new version this week though.